releasenotes/notes/master-96b4f6d34b87fe3d.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - Variables and secrets are now managed in a manner that
+    allows downstream deployers to not have to worry about
+    their settings being overwritten.  They are provided
+    with overrides files that they alone manage after we
+    lay them down.  We have also taken this time to rename
+    our user_* files to separate out what specifically the
+    variables override, whether it be upstream OSA changes
```

Comment:
"splitting the OSA relevant changed of the local RPC-O changes" instead of "OSA changes" (because it's not an osa change, it's an RPC change on OSA scope)